aggregated-translations/ar.js

```diff
@@ -0,0 +1,16 @@
+import { addLocaleData } from 'react-intl';
```

Comment:
Are you wanting to check these in? Based on our conversation earlier we were talking about travis ci or something to build these in production and so you wouldnt need to check these in then?